src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -59,6 +59,7 @@ export class CalcitePickListItem {
   }
 
   /**
+   * @deprecated This will be removed in a future build. Replaced by textLabel.
```

Comment:
[Nitpick] I'd drop `This will be removed in a future build` since it doesn't add much value and users could track it down with the changelog. Sidebar, we could add a general note about deprecated properties in the README . cc @kat10140